lib/ancestry/materialized_path.rb

```diff
@@ -106,10 +106,15 @@ def ancestors?
       def ancestor_ids=(value)
         col = self.ancestry_base_class.ancestry_column
         value.present? ? write_attribute(col, value.join(ANCESTRY_DELIMITER)) : write_attribute(col, nil)
+        @_ancestor_ids = nil
+        @_depth        = nil
+
+        # can't assign to `nil` since `nil` could be a valid result
+        remove_instance_variable(:@_parent_id) if defined?(@_parent_id)
```

Comment:
I guess that "ancestor_ids=" should return assigned value to allow "var = record. ancestor_ids = 'smth'" ?

Reply:
maybe move line 109-113 between 106/107- so the write attribute is returning the right value.

thanks @jgigault